payjoin-cli/src/app.rs

```diff
@@ -154,13 +156,28 @@ impl App {
 
         log::debug!("Enrolled receiver");
 
-        let pj_uri_string =
-            self.construct_payjoin_uri(amount_arg, Some(&enrolled.fallback_target()))?;
+        let amount = Some(Amount::from_sat(amount_arg.parse()?));
+        // convert str to KeyConfig
+        fn decode_ohttp_config(s: &str) -> Result<payjoin::KeyConfig, ()> {
+            let res = bitcoin::base64::decode_config(s, bitcoin::base64::URL_SAFE).unwrap();
+            let res = payjoin::KeyConfig::decode(&res).unwrap();
+            Ok(res)
+        }
+        let ohttp_config = decode_ohttp_config(&self.config.ohttp_config).unwrap();
```

Comment:
if we're going to unwrap there's no reason to make a `Result` returning function. inline is ok